invenio_users_resources/administration/views/users.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio administration users view module."""
+from invenio_administration.views.base import (
+    AdminResourceCreateView,
+    AdminResourceDetailView,
+    AdminResourceEditView,
+    AdminResourceListView,
+)
+from invenio_i18n import lazy_gettext as _
+
+
+class UsersListView(AdminResourceListView):
+    """Configuration for users sets list view."""
+
+    api_endpoint = "/users/sets"
+    name = "USERS"
+    resource_config = "users_resource"
+    search_request_headers = {"Accept": "application/json"}
+    title = "USERS Sets"
+    category = "User management"
+    pid_path = "id"
+    icon = "exchange"
+    template = "invenio_users_resources/users-search.html"
+
+    display_search = True
+    display_delete = True
+    display_edit = True
+
+    item_field_list = {
```

Comment:
The columns should match (when possible) the current ones. Maybe add if it's public or not (and other preferences/profile info)

<img width="1545" alt="Screenshot 2023-06-21 at 14 36 41" src="https://github.com/inveniosoftware/invenio-users-resources/assets/6756943/a29135eb-cdc2-4e2c-bb28-06eabb28a6da">